src/components/calcite-button/calcite-button.tsx

```diff
@@ -52,6 +52,9 @@ export class CalciteButton {
   /** optionally pass an icon to display at the start of a button - accepts calcite ui icon names  */
   @Prop({ reflect: true }) iconStart?: string;
 
+  /** optionally define alignment of button elements. */
```

Comment:
Maybe say button content, since it's not the element itself. Also might be good to note this will only apply to 
width full/half" here

Reply:
Thanks @macandcheese.
Will update.